client/src/lib/notifications/hooks/useReminderNotificationsSetting.spec.ts

```diff
@@ -112,57 +113,79 @@ describe('useNotificationSetting', () => {
 
       useUserState.setState({
         user: {uid: 'some-uid'} as FirebaseAuthTypes.User,
-        userState: {'some-uid': {notificationsEnabled: true}},
+        userState: {'some-uid': {reminderNotifications: true}},
       });
 
       const {result, waitForNextUpdate} = renderHook(() =>
-        useNotificationSetting(),
+        useReminderNotificationsSetting(),
       );
 
       await waitForNextUpdate();
 
       expect(mockGetNotificationSettings).toHaveBeenCalledTimes(1);
-      expect(result.current.notificationsEnabled).toBe(false);
+      expect(result.current.remindersEnabled).toBe(false);
+    });
+
+    it('is undefined if reminderNotifications == undefined', async () => {
+      mockGetNotificationSettings.mockResolvedValueOnce({
+        authorizationStatus: AuthorizationStatus.AUTHORIZED,
+      } as NotificationSettings);
+
+      useUserState.setState({
+        user: {uid: 'some-uid'} as FirebaseAuthTypes.User,
+        userState: {'some-uid': {reminderNotifications: undefined}},
+      });
+
+      const {result} = renderHook(() => useReminderNotificationsSetting());
+
+      expect(mockGetNotificationSettings).toHaveBeenCalledTimes(1);
+      expect(result.current.remindersEnabled).toBe(undefined);
     });
   });
 
-  describe('setNotificationsEnabled', () => {
-    it('requests permission and sets notificationEnabled to true', async () => {
+  describe('setRemindersEnabled', () => {
+    it('requests permission and sets reminderNotifications to true', async () => {
       mockRequestPermission.mockResolvedValueOnce(undefined);
       useUserState.setState({
         user: {uid: 'some-uid'} as FirebaseAuthTypes.User,
       });
 
-      const {result} = renderHook(() => useNotificationSetting());
+      const {result} = renderHook(() => useReminderNotificationsSetting());
 
-      await result.current.setNotificationsEnabled(true);
+      await result.current.setRemindersEnabled(true);
```

Comment:
Don't you get `act` warnings when doing this outside `act`?

Reply:
Hmmm maybe I do... will check!

Reply:
I do... I get a ton of them :D